src/Models/Properties/ElementProperties/DateProperty.cs

```diff
@@ -15,7 +15,9 @@ public partial class BaseProperty
         public string IsFutureDateAfterRelative { get; set; } 
         public string IsFutureDateAfterRelativeType => string.IsNullOrEmpty(IsFutureDateAfterRelative) ? string.Empty : IsFutureDateAfterRelative.Substring(IsFutureDateAfterRelative.LastIndexOf('-') + 1).Trim().ToUpper();  
         public string IsFutureDateBeforeRelative { get; set; } 
-        public string IsFutureDateBeforeRelativeType => string.IsNullOrEmpty(IsFutureDateBeforeRelative) ? string.Empty : IsFutureDateBeforeRelative.Substring(IsFutureDateBeforeRelative.LastIndexOf('-') + 1).Trim().ToUpper();  
+        public string IsFutureDateBeforeRelativeType => string.IsNullOrEmpty(IsFutureDateBeforeRelative) ? string.Empty : IsFutureDateBeforeRelative.Substring(IsFutureDateBeforeRelative.LastIndexOf('-') + 1).Trim().ToUpper();
+        public string IsPastDateBeforeRelative { get; set; }
+        public string IsPastDateBeforeRelativeType => string.IsNullOrEmpty(IsPastDateBeforeRelative) ? string.Empty : IsPastDateBeforeRelative.Substring(IsPastDateBeforeRelative.LastIndexOf('-') + 1).Trim().ToUpper();
```

Comment:
I wonder if it might be useful for readability to use some blank lines here to group things? Thinking maybe:

public string IsFutureDateBeforeRelative { get; set; }
public string IsFutureDateBeforeRelativeType => ...

public string IsFutureDateAfterRelative { get; set; }
public string IsFutureDateAfterRelativeType => ...